src/demo/rewireEnterpriseDataApiService.test.js

```diff
@@ -2,29 +2,68 @@ import EnterpriseDataApiService from '../../src/data/services/EnterpriseDataApiS
 import rewire from './rewireEnterpriseDataApiService';
 
 describe('rewireEnterpriseDataApiService', () => {
-  const { fetchCourseEnrollments, fetchDashboardAnalytics } = EnterpriseDataApiService;
+  const {
+    fetchCompletedLearners,
+    fetchCourseEnrollments,
+    fetchEnrolledLearners,
+    fetchEnrolledLearnersForInactiveCourses,
+    fetchUnerolledRegisteredLearners,
+    fetchCourseEnrollmentsCsv,
+    fetchDashboardAnalytics,
+  } = EnterpriseDataApiService;
 
   afterEach(() => {
     // rewire in the tests overrides the methods on EnterpriseDataApiService so we restore
     // them to their original method calls after each test. Ideally we would just save/restore
     // EnterpriseDataApiService but imports are considered read-only.
+    EnterpriseDataApiService.fetchCompletedLearners = fetchCompletedLearners;
     EnterpriseDataApiService.fetchCourseEnrollments = fetchCourseEnrollments;
+    EnterpriseDataApiService.fetchEnrolledLearners = fetchEnrolledLearners;
+    EnterpriseDataApiService.fetchEnrolledLearnersForInactiveCourses = fetchEnrolledLearnersForInactiveCourses; // eslint-disable-line max-len
+    EnterpriseDataApiService.fetchUnerolledRegisteredLearners = fetchUnerolledRegisteredLearners;
```

Comment:
Have you rebased on `master` recently? The typo in this method name (`fetchUnerolledRegisteredLearners`) is fixed on `master`.